modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/resources/META-INF/resources/ImagePicker/ImagePicker.es.js

```diff
@@ -113,6 +114,8 @@ const ImagePicker = ({
 				<ClayInput.Group>
 					<ClayInput.GroupItem className="d-none d-sm-block" prepend>
 						<ClayInput
+							aria-label="image upload"
```

Comment:
@rijema, will the value of the property `aria-label` be used by a screen reader? If so, then we should consider the language to set the proper attribute's value.